src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,119 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int MIN_AGE = 18;
+    private static final StorageDao storageDao = new StorageDaoImpl();
+    private static final RegistrationService regService = new RegistrationServiceImpl();
+    private User defaultUser;
+
+    @BeforeEach

```

Comment:
```suggestion
    @BeforeAll

```

Reply:
I want to create new test User before every test, that is why I used @ BeforeEach

Reply:
The repository has a static list, if it does not change in your tests, why initialize it every time?

Reply:
@MaksymHorielikov it is more about having a fresh new user for each test in case any test case updates this user fields, so there is no need to change it